src/app/[locale]/(components)/formInput.tsx

```diff
@@ -0,0 +1,31 @@
+import TextInput from "@/app/daisyui/text-input";
+import { ChangeEvent } from "react";
+
+interface formInputProps {
+  value: string;
+  label: string;
+  id: string;
+  type?: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
add prop for `annotation`, which can contain arbitrary text, such as more info or an error, that goes below the form field

Reply:
adds prop with default parameter of empty string. Prop not being used on any fields yet